src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * Class representing task list
+ */
+public class TaskList {
+  public ArrayList<Task> tasks;
```

Comment:
`tasks` could be set to private instead as you already have a getter function below for retrieving it